GitTfs/Core/IGitTfsRemote.cs

```diff
@@ -38,7 +38,6 @@ public interface IGitTfsRemote
         string TfsUsername { get; set; }
         string TfsPassword { get; set; }
         IGitRepository Repository { get; set; }
-        [Obsolete("Make this go away")]
```

Comment:
Yeah, just drop it. I remember having grand plans around this, but I don't remember what they were anymore.